sql/localsql/events/events.go

```diff
@@ -74,3 +81,30 @@ func IterateAllEvents(
 	}
 	return nil
 }
+
+func IterateEventsOps(
```

Comment:
Ditto: This could replace `IterateEvents`. Btw, could you add some tests for it (extend the existing one)?